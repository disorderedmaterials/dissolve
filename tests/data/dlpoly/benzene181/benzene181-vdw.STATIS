Disorder Model                                                                  
 ENERGY UNITS=kjoule/mol 
         1  5.000000E-10        38
  6.734056E+03  2.980000E+02 -1.334653E+03 -1.334653E+03  0.000000E+00
  0.000000E+00  0.000000E+00  0.000000E+00  0.000000E+00  2.959766E+04
  0.000000E+00 -5.245340E+04 -5.245340E+04  0.000000E+00  0.000000E+00
  0.000000E+00  0.000000E+00  0.000000E+00  2.679825E+04  0.000000E+00
  0.000000E+00  0.000000E+00  9.000000E+01  9.000000E+01  9.000000E+01
  0.000000E+00  1.398206E+01  1.574839E-17  1.808866E-16  1.445261E+01
 -2.376485E-01  2.625316E-01 -2.376485E-01  1.346957E+01 -2.157438E-02
  2.625316E-01 -2.157438E-02  1.402399E+01
