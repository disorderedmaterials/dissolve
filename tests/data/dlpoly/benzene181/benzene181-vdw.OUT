
                    DL_POLY Classic 1.9

                              Running on    1 nodes


 ************************************************************************************************************************
 ************************************************************************************************************************
 ************************************************************************************************************************
 ***************     DL_POLY CONTROL file                                                                 ***************
 ************************************************************************************************************************
 ************************************************************************************************************************
 ************************************************************************************************************************


 SIMULATION CONTROL PARAMETERS


 simulation temperature           2.9800E+02

 Nose-Hoover 
 thermostat relaxation time       1.0000E-01

 simulation timestep              5.0000E-10

 selected number of timesteps            1

 real space cut off               1.2000E+01

 real space cut off (vdw)         1.2000E+01

 border width of Verlet shell     5.0000E-01


 Electrostatics : Shifted Coulombic potential

 trajectory file option on  
 trajectory file start                   1
 trajectory file interval             1000
 trajectory file info key                0

 data printing interval                  1

 data stacking interval                  1

 statistics file interval                1

 user allocated job time (d)    ********

 job closure time        (s)    1000.000


SYSTEM SPECIFICATION

 energy units=kjoule/mol 


 number of molecular types               1

 molecular species type                  1


 name of species:             C6H6                                    

 number of molecules                   181

 number of atoms/sites                  12


 atomic characteristics:

                      site     name          mass        charge    repeat    freeze

                         1     CA          12.01100     0.00000         1         0
                         2     HA           1.00800     0.00000         1         0
                         3     CA          12.01100     0.00000         1         0
                         4     HA           1.00800     0.00000         1         0
                         5     CA          12.01100     0.00000         1         0
                         6     HA           1.00800     0.00000         1         0
                         7     CA          12.01100     0.00000         1         0
                         8     HA           1.00800     0.00000         1         0
                         9     CA          12.01100     0.00000         1         0
                        10     HA           1.00800     0.00000         1         0
                        11     CA          12.01100     0.00000         1         0
                        12     HA           1.00800     0.00000         1         0

 number of chemical bonds               12


 chemical bond details:

                            key     index     index                            parameters

                           harm         1         2     0.000000E+00   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         1         3     0.000000E+00   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         1        11     0.000000E+00   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         3         4     0.000000E+00   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         3         5     0.000000E+00   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         5         6     0.000000E+00   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         5         7     0.000000E+00   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         7         8     0.000000E+00   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         7         9     0.000000E+00   1.400000E+00   0.000000E+00   0.000000E+00
                           harm         9        10     0.000000E+00   1.080000E+00   0.000000E+00   0.000000E+00
                           harm         9        11     0.000000E+00   1.400000E+00   0.000000E+00   0.000000E+00
                           harm        11        12     0.000000E+00   1.080000E+00   0.000000E+00   0.000000E+00

 number of bond angles                  18


 bond angle details:

                            key     index     index     index     f-const       angle

                           harm         2         1         3  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         2         1        11  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         3         1        11  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1         3         4  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1         3         5  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         4         3         5  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         3         5         6  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         3         5         7  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         6         5         7  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         5         7         8  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         5         7         9  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         8         7         9  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         7         9        10  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         7         9        11  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm        10         9        11  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1        11         9  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         1        11        12  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000
                           harm         9        11        12  0.0000E+00  120.000000    0.000000    0.000000    0.000000    0.000000

 number of dihedral angles              24


 dihedral angle details:

                            key     index     index     index     index     f-const       angle        trig    1-4 elec     1-4 vdw

                           cos3         2         1         3         4  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         2         1         3         5  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3        11         1         3         4  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3        11         1         3         5  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         2         1        11         9  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         2         1        11        12  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         3         1        11         9  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         3         1        11        12  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         1         3         5         6  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         1         3         5         7  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         4         3         5         6  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         4         3         5         7  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         3         5         7         8  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         3         5         7         9  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         6         5         7         8  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         6         5         7         9  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         5         7         9        10  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         5         7         9        11  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         8         7         9        10  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         8         7         9        11  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         7         9        11         1  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3         7         9        11        12  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3        10         9        11         1  0.0000E+00    0.000000    0.000000    0.500000    0.500000
                           cos3        10         9        11        12  0.0000E+00    0.000000    0.000000    0.500000    0.500000


 number of specified pair potentials         3


                atom 1  atom 2      key                              parameters


                CA      CA        lj       2.92880E-01  3.55000E+00
                CA      HA        lj       1.91735E-01  2.93104E+00
                HA      HA        lj       1.25520E-01  2.42000E+00

 configuration file name: 

          Disorder Model                                                                  


 selected image convention               1


 simulation cell vectors


                        29.925090    0.000000    0.000000
                         0.000000   29.925090    0.000000
                         0.000000    0.000000   29.925090


 system volume            26798.2474300    

  *** warning - : 1..4 scale factors reset for mole    1     sites          3         9

  *** warning - : 1..4 scale factors reset for mole    1     sites          5        11

  *** warning - : 1..4 scale factors reset for mole    1     sites          7         1


 total degrees of freedom                      6513.
 rotational degrees of freedom                    0.
 shell pseudo degrees of freedom                  0.


 sample of starting configuration

      i       x(i)        y(i)        z(i)       vx(i)       vy(i)       vz(i)       fx(i)       fy(i)       fz(i)


      1  9.9443E+00  1.3622E+01 -6.4585E+00  9.4235E+00 -2.3627E+00 -5.0432E+00 -1.5218E+03  5.5780E+03 -6.3348E+03
    110 -1.3173E+01  5.4372E+00 -1.1747E+01 -1.3262E+01 -6.4179E+00  2.1034E+00  3.7415E+03  1.7890E+03 -1.1581E+01
    219 -4.2206E+00 -1.3812E+01 -8.5995E+00 -1.5205E+00 -1.2740E+00  2.4537E+00  7.1828E+03 -2.3089E+04 -1.7624E+03
    328 -5.5143E+00 -7.6332E+00 -8.5263E+00  1.6455E+01  5.2018E+00 -1.2170E+01  4.9785E+03 -4.9283E+03 -7.5575E+03
    437 -1.0963E+01  8.4281E+00  1.2696E+00  6.0151E+00  2.1182E+00  3.9692E-01 -7.3572E+03 -4.1674E+03 -1.5245E+04
    546 -4.4830E+00  1.4575E+01 -1.1030E+01  1.3017E+00 -1.4475E+00  1.7590E+01 -4.0755E+03  5.7037E+02 -3.0360E+03
    655  1.0773E+01  1.4835E+01 -1.1436E+01 -2.9499E+00  1.0003E+00 -1.1543E+01 -1.4157E+03  2.5729E+04  4.0148E+03
    764  9.6974E+00  3.5020E+00 -1.1305E+00 -1.0268E+01 -1.5540E+01 -1.9245E+01 -1.0432E+03  1.6158E+04  9.9030E+02
    873  8.5664E+00 -6.2466E+00  1.1498E+01  5.5452E+00 -5.8281E+00 -2.7761E+00 -5.5342E+03  2.6549E+04  1.0975E+04
    982 -1.9235E+00  5.7123E+00  1.1689E+01 -2.1391E+01 -2.8086E+01 -1.1908E+01 -4.6416E+03  1.3837E+03 -6.6668E+03
   1091  9.7518E+00  7.6085E-01  9.7952E+00 -2.4584E+00  7.5289E+00 -1.3171E+00 -9.8531E+03  4.1077E+03  8.3867E+03
   1200  1.1684E+01 -2.0885E+00 -1.4651E+01 -2.3846E+00  8.4738E+00 -2.1337E+01 -3.4327E+03 -6.8154E+02 -3.9996E+03
   1309 -5.1440E+00 -7.9673E+00 -4.2994E+00  2.9018E+00  2.8447E+00 -9.0888E+00 -5.2714E+03  4.4050E+03 -1.3518E+04
   1418  2.7479E+00  6.3596E+00  1.1444E+01  1.4201E+01  1.1704E+01 -3.3079E+01 -1.2902E+03  9.3438E+03  6.0798E+03
   1527 -5.8097E+00 -1.3087E+01 -1.3713E+01  5.7737E+00  9.8484E-01 -3.9022E+00 -9.6497E+03 -3.7002E+02  2.5935E+04
   1636  1.4163E+01  8.3144E+00 -6.3667E+00  1.3726E+01 -4.0142E+00 -1.6612E+00  6.5661E+03  3.5045E+03 -1.8588E+03
   1745  8.7011E+00  9.5572E+00  2.8305E+00 -6.7994E+00  8.0011E-01  4.9600E+00 -1.1563E+04  7.1693E+03 -1.0979E+03
   1854  1.1247E+01 -1.2575E+01 -4.6799E+00 -1.5536E+01 -3.8328E+01  1.1576E+01 -2.3030E+02  2.6236E+03  4.6026E+02
   1963 -1.4314E+01 -2.9481E+00 -4.8159E+00 -3.9413E+00 -1.5133E+00  1.7128E-01  1.4385E+04  4.2522E+03 -9.2843E+03
   2072  1.1871E+01  9.1332E+00  1.4351E+01 -3.2961E+01 -1.5789E+01 -1.3907E+00  3.7356E+03 -8.6808E+02 -3.2429E+03


long range correction for: vdw energy    -0.182307E+03
                         : vdw pressure  -0.222938E+00

 time elapsed since job start =           0.014 seconds

 ------------------------------------------------------------------------------------------------------------------------

     step     eng_tot    temp_tot     eng_cfg     eng_vdw     eng_cou     eng_bnd     eng_ang     eng_dih     eng_tet
 time          eng_pv    temp_rot     vir_cfg     vir_vdw     vir_cou     vir_bnd     vir_ang     vir_con     vir_tet
 cpu time      volume    temp_shl     eng_shl     vir_shl       alpha        beta       gamma     vir_pmf       press

 ------------------------------------------------------------------------------------------------------------------------
        1  6.7341E+03  2.9800E+02 -1.3347E+03 -1.3347E+03  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
   0.000f  2.9598E+04  0.0000E+00 -5.2453E+04 -5.2453E+04  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
   0.078s  2.6798E+04  0.0000E+00  0.0000E+00  0.0000E+00  9.0000E+01  9.0000E+01  9.0000E+01  0.0000E+00  1.3982E+01

  rolling  6.7341E+03  2.9800E+02 -1.3347E+03 -1.3347E+03  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
 averages  2.9598E+04  0.0000E+00 -5.2453E+04 -5.2453E+04  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
           2.6798E+04  0.0000E+00  0.0000E+00  0.0000E+00  9.0000E+01  9.0000E+01  9.0000E+01  0.0000E+00  1.3982E+01
 ------------------------------------------------------------------------------------------------------------------------


 run terminating. elapsed cpu time =   8.50000E-02, job time = *******d, close time = 1000.00s



 run terminated after       1 steps. final averages calculated over       1 steps.


 ------------------------------------------------------------------------------------------------------------------------

     step     eng_tot    temp_tot     eng_cfg     eng_vdw     eng_cou     eng_bnd     eng_ang     eng_dih     eng_tet
 time          eng_pv    temp_rot     vir_cfg     vir_vdw     vir_cou     vir_bnd     vir_ang     vir_con     vir_tet
 cpu time      volume    temp_shl     eng_shl     vir_shl       alpha        beta       gamma     vir_pmf       press

 ------------------------------------------------------------------------------------------------------------------------
        1  6.7341E+03  2.9800E+02 -1.3347E+03 -1.3347E+03  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
   0.000f  2.9598E+04  0.0000E+00 -5.2453E+04 -5.2453E+04  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
   0.099s  2.6798E+04  0.0000E+00  0.0000E+00  0.0000E+00  9.0000E+01  9.0000E+01  9.0000E+01  0.0000E+00  1.3982E+01

  r.m.s.   0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
 fluctn.   0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
           0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00  0.0000E+00
 ------------------------------------------------------------------------------------------------------------------------


            Approximate 3D Diffusion  coefficients (10^-9 m^2 / s)

            atom        D 
            CA         0.0000E+00
            HA         0.0000E+00


                Average pressure tensor                                       r.m.s. fluctuations 

           1.4453E+01 -2.3765E-01  2.6253E-01                          0.0000E+00  0.0000E+00  0.0000E+00
          -2.3765E-01  1.3470E+01 -2.1574E-02                          0.0000E+00  0.0000E+00  0.0000E+00
           2.6253E-01 -2.1574E-02  1.4024E+01                          0.0000E+00  0.0000E+00  0.0000E+00

            trace/3.   1.3982E+01


 sample of final configuration

      i       x(i)        y(i)        z(i)       vx(i)       vy(i)       vz(i)       fx(i)       fy(i)       fz(i)


      1  9.9443E+00  1.3622E+01 -6.4585E+00  9.4235E+00 -2.3627E+00 -5.0432E+00 -8.2414E+02  3.2854E+03  5.8702E+02
    110 -1.3173E+01  5.4372E+00 -1.1747E+01 -1.3262E+01 -6.4179E+00  2.1034E+00  2.2052E+01  6.8256E+00 -4.5547E+01
    219 -4.2206E+00 -1.3812E+01 -8.5995E+00 -1.5205E+00 -1.2740E+00  2.4537E+00  3.0697E+03 -1.5185E+03  3.2806E+02
    328 -5.5143E+00 -7.6332E+00 -8.5263E+00  1.6455E+01  5.2018E+00 -1.2170E+01  1.4015E+02 -1.2925E+02 -1.5703E+02
    437 -1.0963E+01  8.4281E+00  1.2696E+00  6.0151E+00  2.1182E+00  3.9692E-01 -7.7539E+02  4.7671E+02 -5.1731E+03
    546 -4.4830E+00  1.4575E+01 -1.1030E+01  1.3017E+00 -1.4475E+00  1.7590E+01  2.3263E+02 -8.3119E+02 -5.9151E+01
    655  1.0773E+01  1.4835E+01 -1.1436E+01 -2.9499E+00  1.0003E+00 -1.1543E+01 -1.2220E+03  1.0451E+03  3.1915E+03
    764  9.6974E+00  3.5020E+00 -1.1305E+00 -1.0268E+01 -1.5540E+01 -1.9245E+01 -1.4799E+02 -1.3619E+02 -5.5329E+00
    873  8.5664E+00 -6.2466E+00  1.1498E+01  5.5452E+00 -5.8281E+00 -2.7761E+00  1.0417E+03 -3.2234E+03  6.7879E+02
    982 -1.9235E+00  5.7123E+00  1.1689E+01 -2.1391E+01 -2.8086E+01 -1.1908E+01  4.0221E+02  1.8132E+02 -4.4506E+02
   1091  9.7518E+00  7.6085E-01  9.7952E+00 -2.4584E+00  7.5289E+00 -1.3171E+00 -2.6183E+03  3.1479E+03  9.7913E+02
   1200  1.1684E+01 -2.0885E+00 -1.4651E+01 -2.3846E+00  8.4738E+00 -2.1337E+01 -2.0505E+01 -2.1871E+01 -4.3084E+01
   1309 -5.1440E+00 -7.9673E+00 -4.2994E+00  2.9018E+00  2.8447E+00 -9.0888E+00  3.1441E+03 -8.1990E+02  2.6750E+03
   1418  2.7479E+00  6.3596E+00  1.1444E+01  1.4201E+01  1.1704E+01 -3.3079E+01  1.7475E+01 -6.9246E+01 -1.3522E+02
   1527 -5.8097E+00 -1.3087E+01 -1.3713E+01  5.7737E+00  9.8484E-01 -3.9022E+00 -3.5875E+03 -5.1260E+02 -2.5765E+02
   1636  1.4163E+01  8.3144E+00 -6.3667E+00  1.3726E+01 -4.0142E+00 -1.6612E+00  6.3092E+00 -9.6864E+01 -1.3834E+00
   1745  8.7011E+00  9.5572E+00  2.8305E+00 -6.7994E+00  8.0011E-01  4.9600E+00 -1.1598E+02 -4.3937E+02 -3.0040E+03
   1854  1.1247E+01 -1.2575E+01 -4.6799E+00 -1.5536E+01 -3.8328E+01  1.1576E+01 -2.3695E+01 -3.5471E+01  1.2557E+01
   1963 -1.4314E+01 -2.9481E+00 -4.8159E+00 -3.9413E+00 -1.5133E+00  1.7128E-01  2.2601E+03  3.2735E+01 -9.4550E+02
   2072  1.1871E+01  9.1332E+00  1.4351E+01 -3.2961E+01 -1.5789E+01 -1.3907E+00  5.0465E+02 -1.0439E+02 -5.9233E+01

 time elapsed since job start =           0.100 seconds

